.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "lib/sdl/SDL2_image"]
 	path = lib/sdl/SDL2_image
 	url = https://github.com/libsdl-org/SDL_image.git
+[submodule "lib/usb/ohci_usbhostlib"]
```

Comment:
*(This is a generic comment about commit style, but made on code so we can resolve it more easily)*

Your commit prefix is "USB: ", but nxdk typically uses lowercase prefixes.

Reply:
All should be lowercase now